gg.minion.Minion.yml

```diff
@@ -0,0 +1,75 @@
+app-id: gg.minion.Minion
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+command: minion.sh
+sdk: org.freedesktop.Sdk
+tags:
+  - proprietary
+finish-args:
+  - --env=PATH=$PATH:/app/java/jre/bin:/app/bin
+  - --socket=x11
+  - --share=network
+  - --filesystem=host
+  - --filesystem=~/.var/app/com.valvesoftware.Steam
+  - --filesystem=~/.var/app/net.lutris.Lutris
```

Comment:
this are redundant with the broad "host"

Reply:
I'm guessing that flatpaks don't allow editing of files between themselves unless explicitly allowed so?

Reply:
Same issue is happening when using `filesystem=home`, without these two lines it makes Minion essentially unusuable.